.github/workflows/scripts/setup-mysql.sh

```diff
@@ -7,7 +7,14 @@ if [ "$RUNNER_OS" = "Windows" ]; then
 fi
 
 if [ "$RUNNER_OS" = "macOS" ]; then
-    brew install mysql || true
+    brew install mysql
+
+    cat <<EOF > /opt/homebrew/etc/my.cnf
+[mysqld]
+bind-address = ::1
+mysqlx-bind-address = ::1
+EOF
```

Comment:
DNS resolution changed between Node.js 16 and 18, `localhost` now resolves to `::1` and not `127.0.0.1`, while the Homebrew-installed MySQL listens on IPv4 only by default.